etcdclient/api.go

```diff
@@ -0,0 +1,49 @@
+package etcdclient
+
+import (
+	"log"
+	"os"
+	"time"
+
+	etcd "go.etcd.io/etcd/clientv3"
+	etcdtransport "go.etcd.io/etcd/pkg/transport"
+)
+
+const (
```

Comment:
These "cert_path" items should be passed in the .conf imho.

Reply:
I have made the changes for tlsConfig in a new push.

I will file a Jira to make the change to have "cert_path" be a .conf entry and fix it as my top priority as soon as this merges.  (I want to complete the story so that we can start testing)